### ISSUE TITLE
Stubbing helpers namespace is not possible 

### ISSUE THREAD

**Issue by nazarhussain** [vitaly-t/pg-promise#473]
I was writing some unit tests for the code using `helpers` namespace and found that since the namespace is frozen we can't modify the definition. The `sinon` library is throwing error: 

```
TypeError: Cannot redefine property: insert
```

To generate you can try this code:

```
sinonSandbox.spy(pgp.helpers, 'insert')
```

That caused me for writing test for `insert` helper, but that cause to all functions in `helpers` namespace.

**Comment by vitaly-t** [comment]
This is due to the namespace being read-only.

I'm going to revise the use of read-only namespaces, and probably remove them all, to avoid issues with tests like that.

**Comment by vitaly-t** [comment]
@nazarhussain The fix is now in `master`, and will become available with the next release.

**Comment by vitaly-t** [comment]
Now available in [version 8.1.1](https://github.com/vitaly-t/pg-promise/releases/tag/v.8.1.1).
